src/UserService.Data/Interfaces/ISkillRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.UserService.Models.Db;
+
+namespace LT.DigitalOffice.UserService.Data.Interfaces
+{
+    [AutoInject]
+    public interface ISkillRepository
+    {
+        void Add(DbSkill skill);
```

Comment:
return skill id

Reply:
why?) we know skill id before fnction was called

Reply:
++